src/main/webapp/app/service/trial.service.ts

```diff
@@ -13,8 +13,8 @@ import { environment } from '../environments/environment';
 import { EmailService } from './email.service';
 @Injectable()
 export class TrialService {
-    production = environment.production ? environment.production : false;
-    oncokb = environment.oncokb ? environment.oncokb : false;
+    oncokb = false;
```

Comment:
We should still allow changing the oncokb through the environment, just don't mention it in the example file.

Reply:
I'll roll it back.